### DISCUSSION TITLE
Front end like Vue and React but how about Stencil.js?

### DISCUSSION THREAD

**Discussion by solarsystemic** [adonisjs/core#3638] [General]
Can I use Stencil.js? I wanted to use their Ionic UI and the two work very well.

**Comment by Julien-R44** [comment]
Hello 👋 

I don't know Stencil.JS but it seems to be a classic front-end framework. So I guess the answer is yes.

Have you tried an integration or something ? Are you stuck somewhere? If so, don't hesitate to share your code and we'll help you.
